src/types/const.ts

```diff
@@ -22,8 +22,12 @@ export enum SelectionType {
 
 export const CL_IMPORT_ID = '9c110735ea8b440d8f31c5c68ffb767d'
 
+export const LOCATION_HOST : Readonly<string> = location.hostname
+
+const LOCATION_PORT : string = location.port === '' ? '80' : location.port
```

Comment:
Can you explain the `LOCATION_PORT` change and in what case it was needed?

I had only considered these cases:

local dev - 3000 (or custom port by over ride)
local bot - 3978 (or custom port by over ride)

On, deployed bot's we don't run UI as it's a risk, but you could enable it. on. 

I'm not exactly sure, but on Azure internally it's a string such as "pipe://1234abdf23", and I assume other cloud providers have others, but this gets mapped out to 80 on the external facing URL.

I saw this was about using docker. Does the bot run on 80 in some cases?